src/cryptonote_basic/difficulty.h

```diff
@@ -54,15 +56,15 @@ namespace cryptonote
      */
     bool check_hash(const crypto::hash &hash, difficulty_type difficulty);
 
+    constexpr difficulty_type PULSE_FIXED_DIFFICULTY = 1'000'000 * tools::to_seconds(TARGET_BLOCK_TIME);
```

Comment:
This difficulty is *far* too high and is not easily minable at all.  It should be 1'000'000, not 120'000'000.